prow/oss/config.yaml

```diff
@@ -287,3 +287,15 @@ presets:
   volumeMounts:
   - name: docker-graph
     mountPath: /docker-graph
+- labels:
```

Comment:
Could you also add the PR description as comment above this line?

Reply:
done